dependency/consul_partitions_test.go

```diff
@@ -0,0 +1,65 @@
+package dependency
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	ListPartitionsQuerySleepTime = 50 * time.Millisecond
+}
+
+func TestNewListPartitionsQuery(t *testing.T) {
+	cases := []struct {
+		name string
+		exp  *ListPartitionsQuery
+		err  bool
+	}{
+		{
+			"empty",
```

Comment:
[NIT] were you planning on adding any use cases to this? i don't think you really have to but if this test is not going to describe all/more of the expected ways that NewListPartitionsQuery() is going to set properties, I think we could just remove this test since this empty case is covered in other tests.  Not critical, but just calling that since this was here, I was expecting it to tell me more.  If this test did not exist, I would not ask for it.

Reply:
yeah I'm not particularly sure this test has much value beyond testing that the `New` function returns something usable, I'm okay removing this